test/katex-spec.js

```diff
@@ -1116,6 +1116,31 @@ describe("A markup generator", function() {
     });
 });
 
+describe("A parse tree generator", function() {
+    it("generates a tree", function() {
+        var tree = katex.__parse("\\sigma^2");
+        expect(JSON.stringify(tree)).toEqual(JSON.stringify([
```

Comment:
Yeah - that was my thought as well. The method generates a `"sub": undefined`. I tried leaving it out and adding it in -- it failed in both cases. Not sure :(